vaultwarden/docker-compose.yml

```diff
@@ -8,7 +8,7 @@ services:
       PROXY_AUTH_ADD: "false"
 
   server:
-    image: vaultwarden/server:1.25.0@sha256:f3ebede27f1cf5e78373c3c4a429cf1fdd8d6b13528a2b9ca4fb3cb7cc681ba9
+    image: vaultwarden/server:1.29.1@sha256:89ffbe952338f42265dbff0033d0fa8a88458a52674a05f5ed918875e403dccd
```

Comment:
I think this may be the wrong digest for the multi-arch digest. I get:

```
$ sudo docker pull vaultwarden/server:1.29.1
...
Digest: sha256:c2849f8189e4d425a9d80db0380566cc38577b289b9e6c88330e190e19af5a30
```

Reply:
Please allow me a bit of time to cross-reference and confirm the correct digest. I'll provide an update once I have more information.

Thank you for your patience and vigilance in maintaining the integrity of the processes.

Reply:
You're absolutely right, the correct sha256 digest should be indeed `c2849f8189e4d425a9d80db0380566cc38577b289b9e6c88330e190e19af5a30`. I realized that I mistakenly provided the `SHA256` for the image "`ghcr.io/dani-garcia/vaultwarden:1.29.1`" instead of the correct one, which should have been for "`vaultwarden/server:1.29.1`".

Thank you for bringing this to my attention, and I appreciate your understanding. I'll promptly rectify this mistake.